### DISCUSSION TITLE
Deno support

### DISCUSSION THREAD

**Discussion by amatiasq** [mdx-js/mdx#1674] [General] [answered]
I've been trying to use this library with Deno through esm.sh. There is an issue with it which I reported. Hopefully fixing that issue will make MDX compatible with Deno.

https://github.com/alephjs/esm.sh/issues/167

**Comment by wooorm** [comment]
You could try and use xdm in the meantime, it’s much lighter and doesn’t need babel, and will in a couple of weeks be ported to mdx-js/mdx.
But it seems that CDNs like esm.sh but also skypack are still have problems in the wild, in my experience.

**Reply by amatiasq** [reply]
I was excited by the idea but...
```
> deno eval 'import "https://esm.sh/xdm"'
error: Uncaught SyntaxError: The requested module '/v53/micromark@3.0.5/deno/lib/compile.js' does not provide an export named 'compile'
    at <anonymous> (https://cdn.esm.sh/v53/micromark@3.0.5/deno/lib/preprocess.js:2:8)
```

**Reply by wooorm** [reply]
Yes, this was raised with esm.sh. Broken on their part.

**Comment by ChristianMurphy** [comment, marked as answer]
Trying the latest nightly release and targeting deno specifically, appears to be supported.

```bash
deno eval 'import "https://esm.sh/@mdx-js/mdx@2.0.0-ci.53?target=deno"'
```

**Reply by amatiasq** [reply]
Yes that does the job, thanks!
